R/format_mr_results2.R

```diff
@@ -341,14 +339,15 @@ power_prune <- function(dat,method=1,dist.outcome="binary")
 				r2sum<-sum(r2) # sum of the r-squares for each SNP in the instrument
 				# F<-r2sum*(n-1-k)/((1-r2sum*k )
 				if(dist.outcome == "continuous"){
-					iv.se<- 1/sqrt(unique(dat2$samplesize.outcome)*r2sum) #standard error of the IV should be proportional to this
+					iv.se<- 1/sqrt(mean(dat2$samplesize.outcome)*r2sum) #standard error of the IV should be proportional to this
 				}
 				if(dist.outcome == "binary"){
-					iv.se<-1/sqrt(unique(n.cas)*unique(n.con)*r2sum) #standard error of the IV should be proportional to this
 					if(any(is.na(n.cas)) || any(is.na(n.con))) {
 						warning("dist.outcome set to binary but number of cases or controls is missing. Will try using total sample size instead but power pruning will be less accurate")
-						iv.se<- 1/sqrt(unique(dat2$samplesize.outcome)*r2sum) 
-					}
+						iv.se<- 1/sqrt(mean(dat2$samplesize.outcome)*r2sum) 
+					} else {
+                    	iv.se<-1/sqrt(mean(n.cas)*mean(n.con)*r2sum) #standard error of the IV should be proportional to this
```

Comment:
And same here, I would amend to 

```r
iv.se<-1/sqrt(round(mean(n.cas, na.rm = TRUE))*round(mean(n.con, na.rm = TRUE))*r2sum)
```

Reply:
thanks Tom! I believe I've accepted those changes now. Did it work?

Reply:
It didn't work Philip - I think you probably ticked check boxes to say you'd viewed the comment (or something like) - so I have just made a new release with the `na.rm = TRUE`s in.